src/flatc.cpp

```diff
@@ -216,6 +216,11 @@ int FlatCompiler::Compile(int argc, const char **argv) {
             flatbuffers::PosixPath(argv[argi]));
```

Comment:
you probably didn't intend to touch those files in `scripts/`

Reply:
That was intentional, I ran `chmod +x scripts/*.sh`, so I could check them locally, and I figured that may as well be committed. I can remove this if you prefer though.

Reply:
oh ok, np